cypress/e2e/error.cy.js

```diff
@@ -0,0 +1,70 @@
+describe('Users see error messages if something goes wrong', () => {
+  beforeEach(() => {
+    const kanji = ["s", "f", "g", "v", "b"]
+    kanji.forEach(k => {
+      cy.intercept(
+        "GET",
+        `https://kanjialive-api.p.rapidapi.com/api/public/kanji/${k}`,
+        {
+          statusCode: 200,
+          fixture: `${k}`
+        }
+      ).as('kanji-details')
+    })
+  })
```

Comment:
Nice job with the intercepts! None of them went to the origin when I ran your tests